package.json

```
@@ -18,21 +18,25 @@
     ]
   },
   "scripts": {
-    "build": "babel src/ -d build/",
+    "build": "babel src/ -d build/ --extensions \".js,.ts\" --source-maps inline",
+    "build:types": "tsc --emitDeclarationOnly",
     "prepublish": "yarn run build",
     "test": "jest",
     "flow": "flow",
-    "lint": "eslint '?(__mocks__|src|tests)/**/*.js'",
+    "lint": "eslint \"?(__mocks__|src|tests)/**/*.+(ts|js)\"",
```

Non-blocking OCD comment - this one's different 🕵️‍♀️ :
```suggestion
    "lint": "eslint \"?(__mocks__|src|tests)/**/*.+(js|ts)\"",
```